test/unit/resources/ssh_conf_test.rb

```diff
@@ -35,5 +35,18 @@
         '/etc/ssh/ssh_host_ecdsa_key',
       ]
     end
+
+    it 'check bad path' do
+      resource = load_resource('sshd_config', '/etc/ssh/sshd_config_does_not_exist')
+      _(resource.send(:read_content)).must_equal "Can't find file \"/etc/ssh/sshd_config_does_not_exist\""
+      assert_nil(resource.Protocol)
```

Comment:
We have not used `assert_nil` anywhere in the codebase yet and instead use `must_be_nil`. We should probably be consistent.